### PR TITLE
[docs] Fix wrong test command in CONTRIBUTING

### PR DESCRIPTION
**Pull request by verbose117**
I have removed the `yarn test` command inside the `CONTRIBUTING` guidelines file, which does not exist in the `package.json` of the root folder, but `yarn build` automatically runs tests.

### Related Issues

This PR does not fix any issue, but adds more into the documentation.

#### Tests

- [x] The code changed/added as part of this PR has been covered with tests
- [x] All tests pass locally with `yarn test-unit`

#### Code Review

- [x] This PR has a concise title and thorough description useful to a reviewer
- [x] Issue from task tracker has a link to this PR

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -16,21 +16,22 @@ yarn install
 yarn bootstrap
 yarn build
 yarn lint
-yarn test
```
Comment (styfle) [inline .github/CONTRIBUTING.md:19]: It should be `yarn test-unit`

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -16,21 +16,22 @@ yarn install
...
+Once you are done with your changes (we even suggest doing it along the way), make sure all the test still pass by running:
 
 ```
-yarn build && yarn test
+yarn build
 ```
 
 from the root of the project.
 
+Running `yarn build` automatically runs the tests.
```
Comment (styfle) [inline .github/CONTRIBUTING.md]: This is not true, tests run separately

Comment (verbose117) [reply .github/CONTRIBUTING.md]: ok, but the build ran some tests on my machine.
I am gonna fix that 👍🏻

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (verbose117) [reply .github/CONTRIBUTING.md:19]: ok 👍

**Review by styfle — APPROVED** [review]
Thanks!

**Pull request merged**